### ISSUE TITLE
data augmentation

### ISSUE THREAD

**Issue by LinuxCup** [tusen-ai/SST#95]
whether to use data augmentation? and whether will improve performance when used? Have done ablation study

**Comment by Abyssaledge** [comment]
Yes. You could refer to [this](https://github.com/tusen-ai/SST/blob/main/configs/_base_/datasets/waymo-fsd.py#L31).

**Comment by LinuxCup** [comment]
thiinks for your reply, does data augmentaion is used by default?
